findContact.js

```diff
@@ -0,0 +1,21 @@
+// 13. Write the findContact function that returns a string representing the contact with a given name.
+// If a contact with a given name does not exist, return undefined.
+
+function addContact(object, name, phoneNumber) {
+    return {
+        John: {
+            name: name,
+            phoneNumber: phoneNumber
+        }
+    }
+}
+
+function findContact(object, name) {
+        return `Name: ${object[name].name}, Phone: ${object[name].phoneNumber}`;
```

Comment:
There are some unnecessary spaces here

Reply:
We could make it slightly easier to read by declaring a variable with `object[name]` in the first line